### DISCUSSION TITLE
Problems with Soup

### DISCUSSION THREAD

**Discussion by RoyTM99** [Security-Onion-Solutions/securityonion#6026] [Unsupported Versions] [answered]
Hello Everyone,

What would you recommend to do in the following situation? We currently have Security Onion version 2.3.30 but we need to upgrade to the latest version 2.3.80, this is in part due to the problem described in #4910. However our output to the Internet must be through a proxy, previously we had not configured it and we did it according to the instructions in https://docs.securityonion.net/en/2.3/proxy.html?highlight=proxy however it was not functional, we suspect that it was a problem of the proxy itself so we enabled the full output to the Internet through the Firewall, then we ran the soup command but it shows the following message:

Checking to see if this is a manager.
This is a manager, We can proceed.
Checking to see if this is an airgap install
Found that Security Onion 2.3.30 is currently installed.
Looks like we need access to the upgrade content
If you just copied the .iso file over you can specify the path.
If you burned the ISO to a disk the standard way you can specify the device.
Example: /home/user/securityonion-2.X.0.iso
Example: /dev/sdx1

We put the ISO in the /home/user/ path but we have the following problem:
Something went wrong trying to mount the device.
Ensure you verify the ISO that you downloaded.

The ISO was validated and mounted but without any successful result, as a reference we have searched in our installation evidence and we noticed the detail that we selected the standard option when installing and not the airgap.  In fact our Security Onion solution is not supposed to have internet output but we overlooked this option, could this have affected the performance of soup?
We think that the best option would be to rebuild the console in its entirety since the storage is at full capacity due to the above mentioned problem and the soup command is not fully functional but we would like to know what requirements you recommend for an integration of 52 servers with wazuh agent and 16 devices via Syslog.

We currently have the following resources:
- 10 CPU's
- 64 GB RAM
- /dev - 32 GB
- / - 293 GB
- /nsm - 925 GB

Security Onion: 2.3.30

Thank You
Kind Regards

**Comment by dougburks** [comment, marked as answer]
Yes, it seems like your best option is a fresh installation of the latest version. For hardware requirements, take a look at https://docs.securityonion.net/en/2.3/hardware.html but note that this is going to be heavily dependent on the actual volume of logs from those 52 servers and 16 syslog senders.
